vnext/ReactUWP/Views/DynamicAutomationPeer.cpp

```diff
@@ -129,6 +130,26 @@ winrt::IInspectable DynamicAutomationPeer::GetPatternCore(winrt::PatternInterfac
       (HasAccessibilityState(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityStates::Expanded) ||
        HasAccessibilityState(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityStates::Collapsed))) {
     return *this;
+  } else if (
+      patternInterface == winrt::PatternInterface::Value &&
+
+      // TODO
+      // Should I be limiting this at al? If so, what elements to include apart from this one?
+      accessibilityRole == winrt::PROJECT_ROOT_NAMESPACE::AccessibilityRoles::ComboBox &&
+
+      HasAccessibilityValue(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Text)) {
+    return winrt::make<winrt::PROJECT_ROOT_NAMESPACE::implementation::DynamicValueProvider>(
+        this->try_as<winrt::FrameworkElementAutomationPeer>());
+  } else if (
+      patternInterface == winrt::PatternInterface::RangeValue &&
+      (accessibilityRole == winrt::PROJECT_ROOT_NAMESPACE::AccessibilityRoles::ProgressBar ||
+       accessibilityRole == winrt::PROJECT_ROOT_NAMESPACE::AccessibilityRoles::Adjustable ||
+       accessibilityRole == winrt::PROJECT_ROOT_NAMESPACE::AccessibilityRoles::ScrollBar) &&
+      ((HasAccessibilityValue(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Now) &&
+        HasAccessibilityValue(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Min) &&
+        HasAccessibilityValue(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Max)) ||
+       HasAccessibilityValue(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Text))) {
```

Comment:
>HasAccessibilityValue(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Text [](start = 7, length = 77)

Why this check?  IRangeValueProvider::Value is asking for a numeric value, so it seems like the Now value would satisfy that.